tools/codegen/api/python.py

```diff
@@ -220,11 +324,22 @@ class PythonOutArgument(PythonArgument):
     outputs: Tuple[PythonArgument, ...]
 
     @staticmethod
-    def from_outputs(outputs: Tuple[PythonArgument, ...]) -> Optional['PythonOutArgument']:
+    def from_outputs(outputs: Tuple[PythonArgument, ...], pyi: bool, deprecated: bool) -> Optional['PythonOutArgument']:
         if not outputs:
             return None
 
         size = len(outputs)
+        # TODO: remove HACK
```

Comment:
An example output change after I tested removing this, for `svd` in `_VF.pyi`:

before:
```
def svd(input: Tensor, some: _bool=True, compute_uv: _bool=True, *, out: Optional[Tensor]=None) -> namedtuple_U_S_V: ...
```

after
```
def svd(input: Tensor, some: _bool=True, compute_uv: _bool=True, *, out: Union[Tensor, Tuple[Tensor, ...], List[Tensor]]=None) -> namedtuple_U_S_V: ...
```

The corresponding native_functions.yaml entry for `svd` is [here](https://github.com/pytorch/pytorch/blob/93973ee6993c97c8b9d60c4f720423bc625073ea/aten/src/ATen/native/native_functions.yaml#L6241), which has multiple out arguments. so the `Optional[Tensor]` type hint seems wrong.

Although note that the replacement is a `Union`- I think it should probably be an `Optional[Union]` instead? (see the TODO comment in the code just below here)

Reply:
Yeah, the generated pyi signature is wrong. Will be good to fix 👍